### ISSUE TITLE
Multiple instance mode is not working for cb_adf dsjson

### ISSUE THREAD

**Issue by ataymano** [VowpalWabbit/vowpal_wabbit#1840]
#### Describe the bug
Multiple instances mode is not working for cb_adf dsjson

#### To Reproduce
vw.exe --args args.txt

args.txt:
--cb_adf --dsjson -d file.txt
--cb_adf --dsjson -power_t 0.5 file.txt

#### Expected behavior
Two passes over file.txt with both commandlines

#### Observed Behavior
Error: unrecognised option '--args'

#### Environment
Windows / 8.6.1

**Comment by jackgerrits** [comment]
This issue is fixed by updating line 96 in main.cc
```c++
std::unique_ptr<options_boost_po> ptr(new options_boost_po(l_argc, l_argv));
```

We spoke offline though and you mentioned there are further issues with the driver and multi_ec

**Comment by jackgerrits** [comment]
Would you be able to submit a PR for the partial fix I outlined above? If you'd like I am happy to submit it, just let me know

**Comment by jackgerrits** [comment]
Fixed in #1934
